modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -6,32 +6,54 @@
 = Exposing Elasticsearch as a route
 
 By default, Elasticsearch deployed with cluster logging is not
-accessible from outside the logging cluster. You can enable a re-encrypt route for external
-access to Elasticsearch for those tools that want to access its data.
+accessible from outside the logging cluster. You can enable a route with re-encryption termination 
+for external access to Elasticsearch for those tools that want to access its data.
 
 Internally, you can access Elasticsearch using your {product-title} token, and
 you can provide the external Elasticsearch and Elasticsearch Ops
 hostnames using the server certificate (similar to Kibana).
 
-.Prerequisite
+* The request must contain three HTTP headers:
++
+----
+Authorization: Bearer $token
+X-Proxy-Remote-User: $username
+X-Forwarded-For: $ip_address
+----
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+* Set cluster logging to the unmanaged state.
 
-Set cluster logging to the unmanaged state.
+* You must have access to the project in order to be able to access to the logs. For example:
++
+----
+$ oc login <user1>
+$ oc new-project <user1project>
+$ oc new-app <httpd-example>
+----
 
 .Procedure
 
-. Use the following command to get name of the ElasticSearch pod:
+. Use the following command to get name of the Elasticsearch pod:
 +
 ----
 ESPOD=$( oc get pods -l component=elasticsearch -o name | sed -e "s/pod\///" )
 ----
 
-. Use the following command to extract CA certificate from Elasticsearch:
+. Use the following command to extract the CA certificate from Elasticsearch and write to the *_admin-ca_* file:
 +
 ----
-$ oc extract secret/elasticsearch --to=. --keys=admin-ca
+$oc extract secret/elasticsearch --to=. --keys=admin-ca
```

Comment:
The original line is good.
Note: $ in the first column is meant to be a prompt.

Reply:
@nhosoi Thank you I need to do a global find for this. I think there are more lurking in there.